WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -487,12 +493,14 @@ import WordPressFlux
         }
 
         tableView.tableHeaderView = header
-        if !FeatureFlag.newReaderNavigation.enabled {
             // This feels somewhat hacky, but it is the only way I found to insert a stack view into the header without breaking the autolayout constraints.
-            header.centerXAnchor.constraint(equalTo: tableView.centerXAnchor).isActive = true
-            header.widthAnchor.constraint(equalTo: tableView.widthAnchor).isActive = true
-            header.topAnchor.constraint(equalTo: tableView.topAnchor).isActive = true
+        header.centerXAnchor.constraint(equalTo: tableView.centerXAnchor).isActive = true
+        let headerWidthConstraint = header.widthAnchor.constraint(equalTo: tableView.widthAnchor)
+        if FeatureFlag.newReaderNavigation.enabled {
+            headerWidthConstraint.priority = UILayoutPriority(999)
         }
+        headerWidthConstraint.isActive = true
+        header.topAnchor.constraint(equalTo: tableView.topAnchor).isActive = true
```

Comment:
Probably not a huge deal but it might be better to activate these in bulk since it is supposed to be faster for the autolayout engine:

```suggestion
        let centerConstraint = header.centerXAnchor.constraint(equalTo: tableView.centerXAnchor)
        let topConstraint = header.topAnchor.constraint(equalTo: tableView.topAnchor)
        let headerWidthConstraint = header.widthAnchor.constraint(equalTo: tableView.widthAnchor)
        if FeatureFlag.newReaderNavigation.enabled {
            headerWidthConstraint.priority = UILayoutPriority(999)
        }

        NSLayoutConstraint.activate([
            centerConstraint,
            headerWidthConstraint,
            topConstraint
        ])
```